### PR TITLE
feat: BigTable benchmarking script

### PR DESCRIPTION
**Pull request by hemanshv**
Adds BigTable benchmarking script with Raw stubs.

### CODE REVIEW

**Comment by hemanshv** [comment]
Hi @jskeet ,
I have added the bigtable benchmarking script over here. Please have a look and share your feedback.

Thanks,

File: issues/BigtablePerfBenchmarking/BigtablePerfBenchmarking.csproj [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Google.Cloud.Bigtable.Common.V2" Version="3.0.0" />
```
Comment (jskeet) [inline issues/BigtablePerfBenchmarking/BigtablePerfBenchmarking.csproj:11]: I don't think we should need this reference - it should already be implicit by virtue of having Google.Cloud.Bigtable.V2

**Review by jskeet — APPROVED** [review]
It's *possible* that we'll want to expand this to a solution with multiple projects testing different things in the future, but I think this is fine for now.

File: issues/BigtablePerfBenchmarking/BigtablePerfBenchmarking.csproj (hunk 1 above)
Comment (hemanshv) [reply issues/BigtablePerfBenchmarking/BigtablePerfBenchmarking.csproj]: done

**Pull request merged**